cakefuzzer/domain/vulnerability.py

```diff
@@ -15,6 +15,7 @@ def __hash__(self) -> int:
 
     timestamp: float  # When?
     detection_result: str  # What?
+    detection_location: Optional[str]  # Where?
```

Comment:
Atm the `find_html_location` returns `List[str]` so it should actually be `Optional[List[str]]` 
Or maybe we can get rid of the list, then the `Optional[str]` is ok

Reply:
Parsed ```detection_location``` to str